slashing-protection/src/main/resources/migrations/postgresql/V6__add_db_version.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE database_version (
+    id INTEGER PRIMARY KEY,
+    version INTEGER NOT NULL
+);
+-- Start at version 6, should have previously existed (but now represents migration index).
+INSERT INTO database_version (id, version) VALUES (1, 6);
```

Comment:
does it make sense to keep track of when was the last/latest version updated? if we keep separate records for each update we can insert an auto-date (or am I thinking too much into it).